pytest/tests/loadtest/loadtest.py

```diff
@@ -0,0 +1,70 @@
+from time import sleep, time
+from tqdm import tqdm
+import mocknet_helpers
+import account
+import key
+import base64
+import argparse
+from os.path import join
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Setup loadtest')
+
+    parser.add_argument('--home', type=str, required=True)
+    parser.add_argument('--num_accounts', type=int, default=5)
+    parser.add_argument('--num_requests', type=int, default=50)
+    args = parser.parse_args()
+
+    validator_key = key.Key.from_json_file(join(args.home,
+                                                "validator_key.json"))
+
+    base_block_hash = mocknet_helpers.get_latest_block_hash()
+    nonce = mocknet_helpers.get_nonce_for_key(validator_key)
+
+    my_account = account.Account(validator_key,
+                                 init_nonce=nonce,
+                                 base_block_hash=base_block_hash,
+                                 rpc_infos=[("localhost", "3030")])
```

Comment:
We had issues with `localhost` being interpreted as IPv6. Consider changing to `127.0.0.1` to avoid IPv4-IPv6 confusion.

Reply:
good to know - thanks, fixed